plugins/source/aws/resources/services/resiliencehub/columns.go

```diff
@@ -0,0 +1,44 @@
+package resiliencehub
+
+import (
+	"reflect"
+
+	"github.com/cloudquery/plugin-sdk/schema"
+	"github.com/cloudquery/plugin-sdk/transformers"
+)
+
+var (
+	appARN = schema.Column{
+		Name:            "app_arn",
+		Type:            schema.TypeString,
+		Resolver:        schema.ParentColumnResolver("app_arn"),
+		CreationOptions: schema.ColumnCreationOptions{PrimaryKey: true},
+	}
+	appARNTop = schema.Column{
+		Name:            "app_arn",
+		Type:            schema.TypeString,
+		Resolver:        schema.ParentColumnResolver("arn"),
+		CreationOptions: schema.ColumnCreationOptions{PrimaryKey: true},
+	}
+	assessmentARN = schema.Column{
+		Name:            "assessment_arn",
+		Type:            schema.TypeString,
+		Resolver:        schema.ParentColumnResolver("arn"),
+		CreationOptions: schema.ColumnCreationOptions{PrimaryKey: true},
+	}
+	appVersion = schema.Column{
+		Name:            "app_version",
+		Type:            schema.TypeString,
+		Resolver:        schema.ParentColumnResolver("app_version"),
+		CreationOptions: schema.ColumnCreationOptions{PrimaryKey: true},
+	}
+)
+
+func transformARN(path string) transformers.NameTransformer {
+	return func(field reflect.StructField) (string, error) {
+		if field.Name == path {
+			return "arn", nil
+		}
+		return transformers.DefaultNameTransformer(field)
+	}
+}
```

Comment:
When we create an `arn` field like this we are getting rid of the original column is that really what we want to do rather than just create a new column named `arn`?

Reply:
IIRC in the past we used to rename to `arn`, but I looked at a number of examples now and since the transformer change I think we're leaving the originals in place and adding a new column called `arn`. That's also less of a breaking change, so I'd say let's do that here 👍

Reply:
I just did a quick search and I can't find any place where we use a `NameTransformer` aside from some fixes for `e_c_2` --> `EC2`

Reply:
Yeah. So, I'm either OK to leave the code as it now OR change the PK fields to the proper filed names from the structures (as it'll still be a breaking change), but not just adding a column with the same data.

So, there will be `app_arn` column in `apps` table, as an example of 2nd approach

Reply:
It's actually important that the columns are called `arn`, as that gets used in the AWS resources view. Any tables without `arn` columns get ignored there, so we shouldn't call it `app_arn` if it's the primary `arn` for the table.

I'd say let's be consistent with other tables in the AWS plugin: if other tables are duplicating the data to two columns, which it looks like they are, let's do that here as well

Reply:
updated